packages/graz/src/actions/methods.ts

```diff
@@ -98,3 +99,105 @@ export const sendIbcTokens = async ({
     memo,
   );
 };
+
+export interface InstantiateContractArgs<Message extends Record<string, unknown>> {
+  msg: Message;
+  label: string;
+  fee: StdFee | "auto" | number;
+  options?: InstantiateOptions;
+  senderAddress?: string;
+  codeId: number;
+}
+
+export type InstantiateContractMutationArgs<Message extends Record<string, unknown>> = Omit<
+  InstantiateContractArgs<Message>,
+  "codeId" | "senderAddress" | "fee"
+> & {
+  fee?: StdFee | "auto" | number;
+};
+
+export const instantiateContract = async <Message extends Record<string, unknown>>({
+  senderAddress,
+  msg,
+  fee,
+  options,
+  label,
+  codeId,
+}: InstantiateContractArgs<Message>) => {
+  const { signingClients } = useGrazStore.getState();
+
+  if (!signingClients?.cosmWasm) {
+    throw new Error("Stargate signing client is not ready");
+  }
+  if (!senderAddress) {
+    throw new Error("senderAddress is not defined");
+  }
+
+  return signingClients.cosmWasm.instantiate(senderAddress, codeId, msg, label, fee, options);
+};
+
+export interface ExecuteContractArgs<Message extends Record<string, unknown>> {
+  msg: Message;
+  fee: StdFee | "auto" | number;
+  senderAddress?: string;
+  contractAddress: string;
+}
+
+export type ExecuteContractMutationArgs<Message extends Record<string, unknown>> = Omit<
+  ExecuteContractArgs<Message>,
+  "contractAddress" | "senderAddress" | "fee"
+> & {
+  fee?: StdFee | "auto" | number;
+};
+
+export const executeContract = async <Message extends Record<string, unknown>>({
+  senderAddress,
+  msg,
+  fee,
+  contractAddress,
+}: ExecuteContractArgs<Message>) => {
+  const { signingClients } = useGrazStore.getState();
+
+  if (!signingClients?.cosmWasm) {
+    throw new Error("Stargate signing client is not ready");
+  }
+  if (!senderAddress) {
+    throw new Error("senderAddress is not defined");
+  }
+
+  return signingClients.cosmWasm.execute(senderAddress, contractAddress, msg, fee);
+};
+
+export const getQuerySmart = async <TData>(address?: string, queryMsg?: Record<string, unknown>): Promise<TData> => {
+  const { signingClients } = useGrazStore.getState();
+
+  if (!signingClients?.cosmWasm) {
+    throw new Error("Stargate signing client is not ready");
+  }
+
+  if (address === undefined) {
+    throw new Error("Contract address is undefined");
+  }
+
+  if (queryMsg === undefined) {
+    throw new Error("Query message is undefined");
+  }
+
+  const result = (await signingClients.cosmWasm.queryContractSmart(address, queryMsg)) as TData;
+  return result;
+};
+
+export const getQueryRaw = (keyStr: string, address?: string): Promise<Uint8Array | null> => {
```

Comment:
Let's follow `@cosmjs` params order  
```suggestion
export const getQueryRaw = (address: string, keyStr: string): Promise<Uint8Array | null> => {
```